build/bin/docker-startup.sh

```diff
@@ -31,7 +31,7 @@ function print_servers() {
 # JVM Configuration
 #===========================================================================================
 if [[ "${MODE}" == "standalone" ]]; then
-
+JAVA_OPT="${JAVA_OPT} -XX:+UseConcMarkSweepGC -XX:+UseCMSCompactAtFullCollection -XX:CMSInitiatingOccupancyFraction=70 -XX:+CMSParallelRemarkEnabled -XX:SoftRefLRUPolicyMSPerMB=0 -XX:+CMSClassUnloadingEnabled -XX:SurvivorRatio=8  -XX:-UseParNewGC"
   JAVA_OPT="${JAVA_OPT} -Xms${JVM_XMS} -Xmx${JVM_XMX} -Xmn${JVM_XMN}"
   JAVA_OPT="${JAVA_OPT} -Dnacos.standalone=true"
```

Comment:
Could you please keep same indent, thanks.

Reply:
Oh，that's too bad. I fix it.

Reply:
I looked at it again.I didn't mean to write on that line.so bad. Dockerfile based on jdk8，I want to specify this parameter for both standalone and other